__tests__/app.test.js

```diff
@@ -151,4 +151,50 @@ describe('/api/articles', () => {
 				});
 		});
 	});
+});
+
+describe('/api/articles/:article_id/comments', () => {
+	describe('200: GET:', () => {
+		it('responds with status 200 and an array of comments with the right shape', () => {
+			return request(app)
+				.get('/api/articles/1/comments')
+				.expect(200)
+				.then(({ body: { comments } }) => {
+					expect(comments).toBeInstanceOf(Array);
```

Comment:
Array check is slightly redundant code seeing as the comments data HAS to be an array for the forEach on line 165 to work